pymc3/tests/test_distributions_random.py

```diff
@@ -498,6 +492,33 @@ class TestMoyal(BaseTestDistribution):
     ]
 
 
+class TestSkewNormal(BaseTestDistribution):
```

Comment:
Similarly, we should add a test for the tau parametrization

Reply:
idem.